src/utils/use-constructor-handlers.js

```diff
@@ -6,33 +6,30 @@ const { languages } = config;
 const useConstructorHandlers = () => {
   const [tabsValue, setTabsValue] = useState(0);
   const [imageName, setImageName] = useState('');
-  const [constructorImg, setConstructorImg] = useState('');
+  const [uploadConstructorImg, setUploadConstructorImg] = useState('');
 
   const handleTabsChange = (event, newValue) => {
     setTabsValue(newValue);
   };
 
-  const createConstructor = (values) => {
-    const newConstructorElement = {
-      name: [
-        {
-          lang: languages[0],
-          value: values.uaName
-        },
-        {
-          lang: languages[1],
-          value: values.enName
-        }
-      ],
-      image: values.image,
-      material: values.material,
-      color: values.color,
-      available: values.available,
-      default: values.default,
-      basePrice: +values.basePrice,
-    };
-    return newConstructorElement;
-  };
+  const createConstructor = (values) => ({
+    name: [
+      {
+        lang: languages[0],
+        value: values.uaName
+      },
+      {
+        lang: languages[1],
+        value: values.enName
+      }
+    ],
+    image: values.image,
```

Comment:
You add this image as upload variable in request, and you don't use it. Please remove this

Reply:
fixed